linux_os/guide/services/ssh/ssh_server/sshd_use_strong_macs/oval/shared.xml

```diff
@@ -1 +1,71 @@
-{{{ oval_sshd_config(parameter="MACs", value="((hmac-sha2-512-etm@openssh\.com|hmac-sha2-256-etm@openssh\.com|umac-128-etm@openssh\.com|hmac-sha2-512|hmac-sha2-256|hmac-ripemd160),?)+") }}}
+<def-group>
+  <definition class="compliance" id="sshd_use_strong_macs" version="1">
+    {{{ oval_metadata("Limit the Message Authentication Codes (MACs) to those which are FIPS-approved.") }}}
+    <criteria operator="AND">
+      <extend_definition comment="Installed OS is FIPS certified" definition_ref="installed_OS_is_FIPS_certified" />
```

Comment:
Is this addition expected? The rule didn't previously depend on FIPS certification and the rule.yml doesn't mention anything about FIPS.

Reply:
I saw references to the FIPS-140-2 standard in the CIS spec, dropped it in 98293154c6f0efc802f86ff99dfa87324943183a